components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/event/OAuthEventInterceptor.java

```diff
@@ -226,4 +228,32 @@ default void onPostTokenRevocationBySystem(AccessTokenDO accessTokenDO, Map<Stri
             throws IdentityOAuth2Exception {
 
     }
+
+    /**
+     * This will be called before when Tokens Revoked by application.
+     *
+     * @param revokeRequestDTO {@link OAuthAppRevocationRequestDTO}
+     * @param params           Additional parameters
+     * @throws IdentityOAuth2Exception
+     */
+    default void onPreTokenRevocationByApplication(OAuthAppRevocationRequestDTO revokeRequestDTO,
+                                                   Map<String, Object> params) throws IdentityOAuth2Exception {
+
+    }
+
+    /**
+     * This will be called after when Tokens Revoked by application.
+     *
+     * @param revokeRequestDTO  {@link OAuthAppRevocationRequestDTO}
+     * @param revokeResponseDTO
```

Comment:
incomplete method comment

Reply:
Addressed in [6918b82](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1627/commits/6918b828f239ba766e20f66e6d0af997d891413a)